### ISSUE TITLE
Fusion Reactor won´t ignite

### ISSUE THREAD

**Issue by AHalby** [aidancbrady/Mekanism#4057]
have built 2 different Reactors and loading up the Lasers to 2TRF and launch it with a button, but the reactor will not start. the reactor i built exactly the same as a different players with same settings and same fuel input. his will start everytime and mine will never start.

Glitch?

**Comment by PJB3005** [comment]
Did you put in a loaded Hohlraum?

**Comment by AHalby** [comment]
Yes i did! everything is good, i have 12 Ultimate tank as backup for every fuel source and i let the lasers load up til 2 GRF

**Comment by jburrows82** [comment]
Unless it's changed, default requirement is 400 MJ. Also remember to set your injection rate prior to firing or you won't get a reaction.

You can set the minimum joules required to fire in the laser amp (400 000 000, 400 MJ remember) and walk away while it charges. Setting the min also acts as a failsafe to prevent you firing prematurely.

**Comment by AHalby** [comment]
The Issue isnt that i dont let the amplifier be charged! it can only get so full! it dosent matter if its half full och fully! it will not ignite.

**Comment by jburrows82** [comment]
Like I said, [check your setup](http://wiki.aidancbrady.com/wiki/Fusion_Reactor). Make sure it's correct. Make a creative instance and test your setup and tweak it until it works. If other players are having no problems, then it must be your setup. Not setting the injection rate prior to detonation is a common mistake. Forgetting to put in a charge Hohlraum, also. Not igniting with 400 MJ, etc. Just experiment in a single player creative instance until you figure it out. There are plenty of tutorials and videos to use for research.

"This pen is not writing for me, but other people can write with their pens" leads to diagnostic questions. If your just going to reply "It's not the problem" then this treasure hunt is pointless. You'll have to figure it out on your own.

**Comment by AHalby** [comment]
just build a exact copy in creative and it works like a charm! Wonder if its server issues?

**Comment by bensigal** [comment]
I also can't get the reactor to ignite ever since the server I play on updated to 1.5.1. I did as you suggested and did the exact same thing in creative, and it works. There definitely seems to be a problem with multiplayer servers preventing ignition.

**Comment by AHalby** [comment]
i have a solution now ! 
The Issue is to many output ports! just have 3 input for fuel and then check the "heat" tab it should say 300YRF then it will start, try adding output one at a time. if the "300" dissapears it wont start!

**Comment by jburrows82** [comment]
Post screenshots of your setup, this sounds entirely user error.

**Comment by AHalby** [comment]
Mine works like a charm now! 
it was an issue with output ports.
When i added ports the reactor lost its pre-heat and wouldnt start on ignition.
removed them ports and it works great

**Comment by pyroicedragon** [comment]
I am experiencing a similar anomaly. I have build plenty of reactors in the past (all on 1.7.10), however, now that I have switched to 1.10.2, I can't get my reactor to ignite. I have run through all of the steps mentioned above and even tried reducing the number of ports (though I don't know why that would work) to no avail. There is one difference that I have noticed, however. The heat gauge is displaying the heat in YK. I don't know if this is a change in the mod or a glitch. I would be grateful for any additional suggestions and would be glad to provide more information about my build if it would help.
